src/docs/cookbook/design/tabs.md

```diff
@@ -28,12 +26,11 @@ This recipe creates a tabbed example using the following steps;
 ## 1. Create a `TabController`
 
 For tabs to work, you need to keep the selected tab and content
-sections in sync. This is the job of the
-[`TabController`]({{site.api}}/flutter/material/TabController-class.html).
+sections in sync.
+This is the job of the [`TabController`][].
```

Comment:
This isn't rendering right, but I can't see why. I think it might be fixed when you fix the TabBarView definition (below).